src/microbe_stage/Microbe.cs

```diff
@@ -31,9 +31,10 @@ public class Microbe : RigidBody, ISpawned, IProcessable, IMicrobeAI
 
     private CompoundCloudSystem cloudSystem;
 
+    private List<AudioStreamPlayer3D> otherAudioPlayers = new List<AudioStreamPlayer3D>();
```

Comment:
I think this should be after the `// Child components` comment because this contains child components.

Reply:
Funnily enough I actually moved it up there because I thought the opposite, that it's not actually attached as a child component so I thought it'd be better above.

Reply:
Sound players don't work if they aren't a child component, and you call AddChild with them.